CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs

```diff
@@ -61,5 +61,11 @@ private static DateTimeOffset GetEndingDate(DateTimeOffset currentDateTime)
             var daysToFriday = (7 + (DayOfWeek.Friday - currentDateTime.DayOfWeek)) % 7;
             return currentDateTime.AddDays(daysToFriday);
         }
+
+        private static string TrimEmail(string email)
+        {
+            var indexOfAt = email.IndexOf('@');
+            return email.Substring(0, indexOfAt);
+        }
```

Comment:
`public static string removeEmailDomainName(string email)
        {
            return email.Replace(".co.uk", "")
                .Replace(".com","");
        }`

Would it be safer to check for madetech email without domain name?